src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,48 @@
 import React from 'react';
+import './PostInfo.scss';
+import { FullPost } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: FullPost,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  const hasComment = Boolean(comments.length > 0);
```

Comment:
```suggestion
  const hasComments = Boolean(comments.length);
```
<img width="793" alt="Screenshot 2023-01-30 at 13 22 26" src="https://user-images.githubusercontent.com/94292654/215464208-4dee46ca-efeb-4005-8f20-3bbbfd749a02.png">